### PR TITLE
add hexdump in mqtt log topic and change topics to new topic names

### PR DESCRIPTION
**Pull request by IgorYbema**

### CODE REVIEW

**Comment by MTrab** [comment]
You have forgot to remove the trailing m on powerfull states.

**Comment by MTrab** [comment]
And Quiet_Mode_State_string = "0ff"; is still with a zero in Off :)

**Comment by IgorYbema** [comment]
that is another edit for the states, this is only for topics
but no time today anymore :(

**Comment by MTrab** [comment]
Oh okay

**Comment by IgorYbema** [comment]
For that need to change code for read and write to this standard https://github.com/Egyras/Panasonic-H-Aquarea/blob/master/States.md

File: HeishaMon/HeishaMon.ino [hunk 1]
```diff
@@ -589,93 +589,93 @@ void decode_heatpump_data() {
...
-  float Eproduce = ((float)data[194] - 1.0) * 200;
-  if ( actData["Eproduce"] != Eproduce ) {
-    actData["Eproduce"] = Eproduce;
-    sprintf(log_msg, "received Watt (Elec produced): %.2f", Eproduce); log_message(log_msg);
-    sprintf(mqtt_topic, "%s/%s", mqtt_topic_base, "Eproduce"); mqtt_client.publish(mqtt_topic, String(Eproduce).c_str(), MQTT_RETAIN_VALUES);
+  float Energy_Production = ((float)data[194] - 1.0) * 200;
+  if ( actData["Energy_Production"] != Energy_Production ) {
+    actData["Energy_Production"] = Energy_Production;
+    sprintf(log_msg, "received Watt (Energy_Production): %.2f", Energy_Production); log_message(log_msg);
+    sprintf(mqtt_topic, "%s/%s", mqtt_topic_base, "Energy_Productions"); mqtt_client.publish(mqtt_topic, String(Energy_Production).c_str(), MQTT_RETAIN_VALUES);
```
Comment (MTrab) [inline HeishaMon/HeishaMon.ino]: Small typo - a S too much in Energy_Production

Comment (IgorYbema) [reply HeishaMon/HeishaMon.ino]: good one

**Pull request merged**